### PR TITLE
Add the Text "API" to the Learn Page 

### LINKED ISSUE #257: Need a Way to Search for API Docs in the Learn Page
**Description:**
Currently, it is unable to do a Ctrl+f and find API Docs in the Learn page as the word "API Docs" is not added in it as shown below.

Screenshot 2020-02-26 at 14 23 48

Therefore, we can add the text "API Docs" so that it can be searched easily.

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This adds the text "API Docs" so that it can be searched easily.
> Fixes #257 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ redirect_from:
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Learn the “batteries-included” standard library.</p>
+     <p class="cBoxpara">Learn the “batteries-included” in the standard library APIs.</p>
```
Comment (MaryamZi) [inline learn.md]: Can't we say
```suggestion
     <p class="cBoxpara">Learn the APIs of the “batteries-included” standard library.</p>
```
instead?

It is the standard library itself that is "batteries-included" IMO, and saying "in the standard library APIs" doesn't seem to read right.

Comment (praneesha) [reply learn.md]: Saying both "APIs" and "“batteries-included” doesn't sound right, IMO. Therefore, changed to "Learn the “batteries-included” standard library (APIs)."

Comment (MaryamZi) [reply learn.md]: "batteries-included" is like an adjective for the standard library. So, IMO, if "Learn the APIs of the standard library" reads OK, "Learn the APIs of the "batteries-included" standard library" is also OK? WDYT?

The issue I see with the current suggestion with (APIs) at the end is that it seems disconnected.

It's like we're saying learn the standard library and then just adding "(APIs)" at the end.

Comment (praneesha) [reply learn.md]: Nope. Saying  "Learn the APIs of the "batteries-included" standard library" distorts the meaning as the included batteries are the APIs themselves.

@Samallama - WDYT?

Comment (MaryamZi) [reply learn.md]: I don't agree. IMO, batteries-included is used here to say that the standard library itself comes with a lot of functionality, so you wouldn't have to look elsewhere.

I believe it is the standard library as a whole that is batteries-included, not just the APIs.

Another suggestion,

"Check out the API docs of the batteries-included standard library."

Comment (praneesha) [reply learn.md]: +1 to say ""Check out the API docs of the batteries-included standard library."

@Samallama - Please confirm.

Comment (Samallama) [reply learn.md]: +1

Comment (MaryamZi) [reply learn.md]: I guess it should be

```md
Check out the API docs of the “batteries-included” standard library.
```

Comment (praneesha) [reply learn.md]: Added that.

**Pull request merged**